package NKikimrStoragePool;
option java_package = "ru.yandex.kikimr.proto";

message TStoragePool {
    optional string Name = 1;
    optional string Kind = 2;
}

message TChannelBind {
    //optional uint32 Channel = 1; // it should be equal to array index, so it doesn't make any sense to have one
    optional string StoragePoolName = 2;
    optional string StoragePoolKind = 3;
    optional float IOPS = 10;
    optional uint64 Throughput = 11;
    optional uint64 Size = 12;
    optional bool PhysicalGroupsOnly = 20; // default: true for channels 0, 1 and false for other channels
}
